propagation/onboarding.md

```diff
@@ -0,0 +1,76 @@
+# Algorithm Onboarding Service
+
+APEx offers an algorithm onboarding service as part of its propagation services to ensure that project results remain
+accessible to the EO community beyond the project's completion. This service aims to sustain and promote the use of
+algorithms developed during ESA projects, ensuring their longevity and continued utilization.
+
+For engagement with stakeholders of the algorithm, APEx collaborates closely with
+the [Stakeholder Engagement Facility](https://esa-sef.eu/). This initiative will provide outreach and training towards
+users, to maximize the uptake of the onboarded algorithms.
+
+By leveraging APEx’s Algorithm Onboarding service, you can ensure that your project’s innovations continue to benefit
+the
+EO community long after your project ends.
+
+## Service Overview
+
+### Hosting Platform Onboarding Support
+
+APEx supports your project in selecting the appropriate hosting environment for your algorithm based on
+the [APEx Interoperability and Compliance Guidelines](../interoperability/algohostingenv.md). APEx also assists in the
+onboarding process, ensuring that your algorithm is available for execution on the selected platform and ready for
+further integration into the APEx Algorithm Catalogue.
+
+:::{.callout-tip}
+A full list of the known APEx-compliant algorithm hosting platforms is available [here](index.qmd#what-are-the-platforms-to-choose-from).
+:::
+
+### Automated Testing and Benchmarking
+
+To maintain the integrity of the algorithms, APEx conducts automated testing and benchmarking in collaboration with the
+respective project teams. This ensures that the algorithms remain functional and accessible over time. APEx offers this testing service for free, which reduces the maintenance burden on projects.
```

Comment:
please notice that a hosted service in a platform (and particularly a service registered in the NOR) needs to follow the contract rules and what is agreed in the SLA between the platform and ESA
also I don't see how the testing a service can be assured for free when it implies the use processing resources

Reply:
The NoR does not currently check any claims made in the SLA, so we can't rely on it unfortunately. Testing may require that APEx has platform subscriptions, sponsored via NoR, so from a service provider or user perspective, it's 'for free'.
I don't fully understand the argument against invoking services. As a service provider that offers useful and working services, why should I be afraid that someone actually uses it? 
It's also not that APEx triggers random or 'difficult' tests. The test scenarios or chosen by the service provider, so costs are predictable, and difficult cases can be avoided if desired.

Reply:
that is the purpose of the SLA of a platform, and this it is not a issue of random or difficulty, it is an issue that every request needs to be payed 
if during a month, you ask to execute a request every hour or day, you need a subscription and an authorisation with enough credits

in a separate issue ESA is asking to draft the NoR request so we can follow this approach 

https://github.com/ESA-APEx/apex_algorithms/issues/28

Reply:
The frequency of invoking algorithms is up for discussion. I would envision to make it configurable per scenario.
A good default would be somewhere in the weekly/biweekly range for algorithms that are sufficiently cheap. For more expensive ones, we could go for monthly or even less frequent.

If we take the example of a more expensive openEO algorithm of 1€ per scenario run, then with biweekly frequency it costs us 26€ per year. So if we have a nice catalog of 100 algorithms like this, the cost would be 2600€, which seems affordable.

Reply:
Please take a look at the conversation below as a suggestion was made to update the text.